### PR TITLE
Fixed wing guide

### PR DESCRIPTION
**Pull request by synixebrett**

### CODE REVIEW

File: guides/roles/fixed-wing.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+3. LOAL-LOW | Missile immediately gains ~90m altitude
+4. LOAL-HI | Missile immediately gains ~300m altitude
+
+LOBL is only available when the LOBL status indicator is lit red.
+
+![Diagram](img/hellfire_lock.png)
+
+### Radar
+
+Radar functions similarly to Laser / IR Guided (Propelled), but requires passive radar or active radar to be enabled. Active radar is toggled with `<Control> + R`
```
Comment (nameless) [inline guides/roles/fixed-wing.md]: This image might be useful for [RADAR](http://i.imgur.com/IPaFwup.png).

File: guides/roles/fixed-wing.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+This guide will only cover the use of NATO equipment.
+
+<hr/>
+
+# Basics
+
+This guide will not cover the basics of flight, and only covers operations.
+
+## HMD
```
Comment (nameless) [inline guides/roles/fixed-wing.md]: [HMD =/= HUD](https://www.sygic.com/blog/2017/what-is-a-head-up-display-a-k-a-hud)

Comment (nameless) [inline guides/roles/fixed-wing.md]:
```suggestion
## Heads-up display (HUD)
```

File: guides/roles/fixed-wing.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+![Diagram](img/hmd.png)
```
Comment (nameless) [inline guides/roles/fixed-wing.md]: rename to hud.png please

File: guides/roles/fixed-wing.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+- Mk82  
+  500lb, unguided
+- GBU-12  
+  500lb, laser guided, high explosive
+
+## Targeting
+
+### Unguided (Propelled)
+
+The estimated location of impact and time of flight in seconds are indicated in the HMD.
```
Comment (nameless) [inline guides/roles/fixed-wing.md]:
```suggestion
The estimated location of impact and time of flight in seconds are indicated in the HUD.
```

File: guides/roles/fixed-wing.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+![Diagram](img/unguided_estimate.png)
+
+### Unguided
+
+The estimated location of impact and the time of flight in seconds are indicated in the HMD.
```
Comment (nameless) [inline guides/roles/fixed-wing.md]:
```suggestion
The estimated location of impact and the time of flight in seconds are indicated in the HUD.
```

File: guides/roles/fixed-wing.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+![Diagram](img/bomb_hmd.png)
```
Comment (nameless) [inline guides/roles/fixed-wing.md]: rename to hud.png please

File: guides/roles/fixed-wing.md (hunk 1 above)
Comment (synixebrett) [reply guides/roles/fixed-wing.md]: https://github.com/Synixe/Documents/blob/fixed_wing/guides/roles/img/sensor.png

Alright using it eh

File: guides/roles/fixed-wing.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+
+The estimated location of impact and the time of flight in seconds are indicated in the HUD.
+
+![Diagram](img/bomb_hud.png)
+
+This is obviously a bad idea when using a bomb, as it requires the pilot to be low or diving. The estimated location can also be seen from the underbelly camera marked by a small crosshair.
+
+![Diagram](img/bomb_crosshair.png)
+
+If the location of impact is obstrcuted the crosshair will be overlayed with a cross.
```
Comment (nameless) [inline guides/roles/fixed-wing.md]:
```suggestion
If the location of impact is obstructed the crosshair will be overlayed with a cross.
```

Comment (nameless) [inline guides/roles/fixed-wing.md]:
```suggestion
If the location of impact is obstrcuted the crosshair will be overlaid with a cross.
```

**Pull request merged**